Sources/Public/Animation/LottieAnimationView.swift

```diff
@@ -1374,8 +1374,13 @@ final public class LottieAnimationView: LottieAnimationViewBase {
         // `playFrom` time to be the `currentFrame`. Since the animation duration
         // is based on `playFrom` and `playTo`, this automatically truncates the
         // duration (so the animation stops playing at `playFrom`).
+        //  - Don't do this is the animation is already at that frame
```

Comment:
```suggestion
        //  - Don't do this if the animation is already at that frame
```

Reply:
ty

Reply:
Xcode has spell check but what I really need seems to be grammar check